ai_guide/clients/open_ai_client.py

```diff
@@ -3,28 +3,26 @@
 
 import openai
 from dotenv import load_dotenv
-from rest_framework import serializers
 
-from api_attractions.consts import MAX_TOKENS, MESSAGE, SYSTEM_MSG, TEMPERATURE
 from attractions.classes import AttractionInfo
-from open_ai_client.serizalizers import AttractionInfoSerializer
+from clients.attraction_info_serializer import AttractionInfoSerializer
+from clients.config import MAX_TOKENS, MESSAGE, MODEL, SYSTEM_MSG, TEMPERATURE
 
 load_dotenv()
 
 OPEN_AI_API_KEY = os.getenv('OPEN_AI_API_KEY')
 
 
 class OpenAiClient:
-    MODEL = 'gpt-3.5-turbo'
 
     def __init__(self) -> None:
         openai.api_key = OPEN_AI_API_KEY
 
-    def get_answer(self, query: str) -> AttractionInfo:
+    def get_answer(self, query: str) -> (bool, AttractionInfo or str):
```

Comment:
I think it's overcomplicated in terms of returned value.

I can just return `AttractionInfo | None` and check in the outer code:
```
if attraction_info:
  proceed_with_attraction_info()
else:
  do_error_workaround()
```